backend/deploy/e2e.sh

```
@@ -40,6 +42,10 @@ say()  { printf '\n%s==> %s%s\n' "$DIM" "$*" "$OFF"; }
 pass() { checks=$((checks + 1)); printf '%s  ✓%s %s\n' "$GREEN" "$OFF" "$*"; }
 die()  { printf '%s  ✗ %s%s\n' "$RED" "$*" "$OFF" >&2; exit 1; }
 
+#: 埠打錯的話，錯誤會晚 120 秒才以「admission 沒起來」的樣子出現 —— 在這裡擋掉。
+[[ $ADMISSION_HOST_PORT =~ ^[0-9]+$ ]] && ((ADMISSION_HOST_PORT >= 1 && ADMISSION_HOST_PORT <= 65535)) \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)